terraform/modules/cloudwatch/variable.tf

```diff
@@ -0,0 +1,30 @@
+variable "resource_prefix" {
```

Comment:
from previous comment we should have something to determine the target type like this:

variable "target_type" {
  type        = string
  description = "Select a target type for the cloudwatch event"
  sensitive   = false
  validation {
    condition     = contains(["ecs"], var.target_type)
    error_message = "The variable target_type must be one of: ecs"
  }
}